packages/sender/src/lib/sender.ts

```diff
@@ -278,6 +278,20 @@ const Sender: WalletBehaviourFactory<InjectedWallet> = async ({
           return res.response;
         });
     },
+
+    async importAccountsInSecureContext({ accounts }) {
+      if (window.near && window.near.isSender) {
+        window.near.batchImport({
+          keystore: JSON.stringify(accounts),
```

Comment:
The PR calling this method expects data as an [array of arrays](https://github.com/near/near-wallet/pull/2957/files#diff-594a571a3658085ce173558b02d6f1a52304c8116640d608beb737e60883a5c3R90) so we can just map it to the legacy format here for now.

If we haven't already we'll need to reach out to Sender about the data no longer being encrypted; we can start the conversation on that linked PR. This implementation won't work until that change is made at the very least, but they may also want to change the `keystore` parameter name and potentially match our interface of an array of objects for each account ID/private key pair.

Reply:
@hcho112 I was thinking we could pass in the format that is currently expected by `batchImport`, but let's get clarification from Sender Wallet either way before we decide on an implementation. That way we don't end up doing it twice 😄